cli/fmt.rs

```diff
@@ -160,11 +162,10 @@ pub async fn format(args: Vec<String>, check: bool) -> Result<(), ErrBox> {
   }
   let config = get_config();
   if check {
-    check_source_files(config, target_files).await?;
+    check_source_files(config, target_files).await
   } else {
-    format_source_files(config, target_files).await?;
+    format_source_files(config, target_files).await
```

Comment:
By the way and FWIW, the first time I saw this say "Formatted 0 files" I got confused and thought it didn't find any files to format. It might be good to change it to not to show anything in the output if no files are formatted.